packages/ckeditor5-typing/src/inserttextcommand.ts

```diff
@@ -116,9 +98,28 @@ export default class InsertTextCommand extends Command {
 }
 
 export interface InsertTextCommandOptions {
+
+	/**
+	 * The text to be inserted.
+	 */
 	text?: string;
+	// TODO note that those 2 options are exclusive (either selection or range)
```

Comment:
Who is the adressee of this TODO? Did you forget something or do we have to change something in the future.

`(either selection or range)` - it's `Selection | DocumentSelection`.